features/step_definitions/user_steps.rb

```diff
@@ -2,7 +2,7 @@
 end
 
 When(/^I visit the "(.*?)" page$/) do |arg1|
-  visit arg1  
+  visit arg1
 end
```

Comment:
I would go ahead and change this from `arg1` to `page` or something more helpful